src/configs/index.js

```diff
@@ -0,0 +1,54 @@
+import ImportContactsIcon from '@material-ui/icons/ImportContacts';
+
+const languages = ['uk', 'en'];
+
+const routes = {
```

Comment:
Please, remove these variables and move its values to the object `config` where it is used

Reply:
Removed!